<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Owners</class>
 <widget class="QFrame" name="UC_Owners">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Frame_Owners">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Frame_Owners_Info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tbl_owners_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>1024</width>
       <height>562</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>الإسم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الهاتف الاساسى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الهاتف الثانى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>العنوان</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>البريد الالكترونى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاجر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>عميل</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_edit_owner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbxIsClient">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>من العملاء</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_273">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>هاتف ثانى</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>ألاسم</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_owner">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerSearch">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>190</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-right:10%;</string>
     </property>
     <property name="placeholderText">
      <string>بحث سريع بالإسم</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_owner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbxIsTrader">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>تاجر</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerAddress">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>العنوان</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البريد الإلكترونى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerMail">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxClientsNames">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>251</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerPhone1">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>هاتف أساسى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerPhone2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Owners_Reports">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_owners_frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>بيانات الملاك</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_owners_archieve_frame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>أرشيف الملاك</string>
    </property>
   </widget>
   <zorder>Frame_Owners_Reports</zorder>
   <zorder>Frame_Owners_Info</zorder>
   <zorder>btn_owners_frame</zorder>
   <zorder>btn_owners_archieve_frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
